packages/amplify-codegen/tests/commands/models.test.js

```diff
@@ -19,10 +19,13 @@ const MOCK_CONTEXT = {
     },
     getProjectConfig: jest.fn()
   },
+  usageData: {
+    version: '0.0.0'
+  }
 };
 const OUTPUT_PATHS = {
-    javascript: 'src',
-    android: 'app/src/main/java',
+    javascript: 'src/models',
+    android: 'app/src/main/java/com/amplifyframework/datastore/generated/model',
```

Comment:
Why are these paths changed? 
I don't see any change in the model output path fetch in "getModelOutputPath(context)"

Reply:
These point to the actual folders where models are generated.